docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,330 @@
-generic Linux and macOS systems using Intel and GNU compilers. Four `levels of support
-<https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_
-have been defined for the SRW Application, including pre-configured (level 1), configurable
-(level 2), limited test platforms (level 3), and build only platforms (level 4). Each
-level is further described below.
-
-For the selected computational platforms that have been pre-configured (level 1), all the
-required libraries for building the SRW Application are available in a central place. That
-means bundled libraries (NCEPLIBS) and third-party libraries (NCEPLIBS-external) have both
-been built. The SRW Application is expected to build and run out of the box on these
-pre-configured platforms and users can proceed directly to the using the workflow, as
-described in the Quick Start (:numref:`Chapter %s <Quickstart>`).
-
-A few additional computational platforms are considered configurable for the SRW
-Application release. Configurable platforms (level 2) are platforms where all of
-the required libraries for building the SRW Application are expected to install successfully,
-but are not available in a central place. Applications and models are expected to build
-and run once the required bundled libraries (NCEPLIBS) and third-party libraries (NCEPLIBS-external)
-are built.
-
-Limited-Test (level 3) and Build-Only (level 4) computational platforms are those in which
-the developers have built the code but little or no pre-release testing has been conducted,
-respectively. A complete description of the levels of support, along with a list of preconfigured
-and configurable platforms can be found in the `SRW Application wiki page 
-<https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_.
-
-User Support, Documentation, and Contributing Development
-=========================================================
-
-A forum-based, online `support system <https://forums.ufscommunity.org>`_ with topical sections
-provides a centralized location for UFS users and developers to post questions and exchange
-information. The forum complements the formal, written documentation, summarized here for ease of
-use.  
+----------------------------
+
+The SRW Application has a portable CMake-based build system that packages together all the components required to build the SRW Application. Once built, users can generate a Rocoto-based workflow that will run each task in the proper sequence (see `Rocoto documentation <https://github.com/christopherwharrop/rocoto/wiki/Documentation>`__ for more on workflow management). Individual components can also be run in a stand-alone, command line fashion. 
+
+The SRW Application allows for configuration of various elements of the workflow. For example, users can modify the parameters of the atmospheric model, such as start and end dates, duration, time step, and the physics suite for the forecast. 
+
```

Comment:
"the physics suite for the forecast"... I might say "the physics suite used for the simulation"